src/number/index.js

```diff
@@ -1,6 +1,9 @@
 // @flow
 import sample from '../sample';
 
-export default function number (): number {
-  return Math.random() * 1000000 * sample([1, -1]);
+export default function number (
+  lowerInclusive: number = 1000000,
```

Comment:
did you mean to make this `-1000000`? if I'm reading this right calling `fake.number()` with no arguments will always generate exactly `1000000` with this setup, no?

Reply:
🤦‍♂️  yes, yes I did